.travis.yml

```diff
@@ -83,6 +83,35 @@ stages:
 # using it with stages and some variables/sections cannot be overridden.
 jobs:
   include:
+    # Wheel builds
+    - sudo: required
+      services:
+        - docker
+      env:
+        - CIBW_BEFORE_BUILD="pip install -U Cython"
+        - CIBW_SKIP="cp27-* cp34-*"
+        - TWINE_REPOSITORY_URL=https://test.pypi.org/legacy/
```

Comment:
Do you think that we should be pointing to the test pypi servers and try this for a while to make sure that everything works as expected? ... or you are very confident about this :)

Reply:
My thinking was that we'd use test.pypi.org as mostly a data store so we can download and test the artifacts before publishing a release to the production pypi. While I'm sure the full automatic approach would work fine, having the manual step in the middle is nice to just make sure it all worked as expected.